lib/active_model/serializer/adapter/json.rb

```diff
@@ -4,6 +4,11 @@ module ActiveModel
   class Serializer
     class Adapter
       class Json < Adapter
+        def initialize(serializer, options = {})
+          super
+          serializer.root = true if @options[:root]
```

Comment:
this logic doesn't make sense to me.  why are we only checking if the value of options[:root] is truthy?